stimuli.js

```diff
@@ -51,18 +51,3 @@ const TEST_ITEMS = [
 function getPracticeItems() {
```

Comment:
Perhaps it is best to move the getPracticeItems() to index.html as well, Sorry about that, I could have said that earlier. This makes stimuli.js almost purely devoted to tables that describe the stimuli.